scripts/generate.rb

```diff
@@ -3,7 +3,9 @@
 
 for each_dir in Dir["source/languages/**"]
     exit_code = Process.wait(Process.spawn("ruby", each_dir+"/generate.rb"))
-    if exit_code != 0
```

Comment:
Why was this changed?

Reply:
Actually I could use some help on this section. 
There's this really strange bug. When using backticks in ruby or running the command in the command line, I get exit code `0` for the `cpp/generate.rb`.

However, the same command using `system()` or `Process.spawn()` will return a `76XXX` error code like `76101` and every time I run it the error code changes even though the process finishes successfully.

Reply:
`Process.wait` returns the process id of the waited on process. use `$?` to get the exit code.

Reply:
Haha thanks, I should've caught that.